rfcs/0098-community-team.md

```diff
@@ -0,0 +1,194 @@
+
+  The community team may decide to reach out to active community members they think would be a good fit for the team, if the member has not already applied to the team. The community team should keep internal records of what such outreach has been done so as to not duplicate efforts or reach out to members who have expressed disinterest prior.
+
+  The team's focus should be on developing leadership from within, not on bringing in experienced people from outside. This is important to maintaining ideological continuity.
+
+2. Community team members may resign at any time. Such resignations are effective immediately. In exceptional circumstances, members may also be removed by decision of the team; it is hoped that this will never be needed.
+
+3. The community team will decide on new members by consensus. That is, new members will be added to the team if and only if every then-current team member is in favor of adding them. In the event of removing members, the team will decide by a 2/3 majority vote, with the permissible votes being "yes" and "no". All team members are required to participate in membership decisions.
+
+  The rationale for using consensus as the basis of adding members is that it will lead to conservative decisions, which is expected to help with the goal of ideological continuity. This will also make it harder for a hostile takeover to succeed. Removing members is slightly easier than adding them, for the same reasons.
+
+  The formal decision-making process is intended only as a way to resolve membership questions and other long-term questions about the direction of the team; it is neither required nor recommended for routine moderation decisions.
+
+4. The team should attempt to maintain a sufficient size such that there are enough members to cover all the community spaces without major gaps. On no account should the team ever be so large that members can't all know each other, or can't keep each other in the loop on decisions.
+
+5. Members are expected to remain in regular communication with each other and develop their social bonds; it is understood that social bonds among team members are vitally important to doing the job properly. The arrival of new members should be celebrated, and the team should maintain a low-latency communication channel where members can easily reach each other for discussions that are private from the community at large. Retired team members should leave that channel.
+
+6. There will be a formal ceremony to onboard new members and graduate retiring members. Ceremonies will take place on video calls attended by all current team members and any interested members of the community. Nobody will be required to show their face or voice, but team members must listen and actively participate, even if only through text.
```

Comment:
Huh. Okay. Thank you for that. It's very fair. That definitely deserves some thought.

Reply:
Strongly agree with @ToxicFrog's concern here. Auditory issues are pretty widespread. For me personally, a single such call basically costs me a day's worth of energy simply because I have difficulties understanding what people are saying over a VoIP link - which effectively means I just cannot do calls.

Is there any particular reason to specify this so explicitly in the RFC to begin with?